docs/protocol.rst

```diff
@@ -44,13 +44,13 @@ payload.  All data transmitted via |MHP| are split into chunks.
 
 **Channel** refers to a set of chunks transmitted over the same connection,
 identified among other ones using a number that is unique throughout the
-currently active channels connection-wise.  Channels provide multiplexing
+currently active channels connection-wise.  Channel provides multiplexing
```

Comment:
Not sure about this change because "connections" is plural further in the sentence, and connection ⊂ channels.